time/test/time.ts

```diff
@@ -346,6 +372,54 @@ describe('@cycle/time', () => {
             Time.run(complete);
           });
 
+          it('prints primitive values', done => {
+            // most's 'merge' operator seems to be broken with Time.diagram() :
+            // it throws "cannot read property 'dispose' of undefined (on MergeSink.prototype.end)"
```

Comment:
I had to skip this test for most.js because it kept on failing on : 

```js
most.merge(
  Time.diagram('--')
)
```

With a runtime type error `cannot read property 'dispose' of undefined`